### PR TITLE
Add button to toggle visibility of the Flutter frames chart

### PR DESCRIPTION
**Pull request by kenzieschmoll**
show-hide-frames

TODO: animate showing / hiding the chart - https://github.com/flutter/devtools/issues/4576

### CODE REVIEW

**Review by CoderDake — APPROVED** [review]
Couple small things.
LGTM otherwise

File: packages/devtools_app/lib/src/shared/common_widgets.dart [hunk 1]
```diff
@@ -461,6 +502,41 @@ class CollapseAllButton extends StatelessWidget {
   }
 }
 
+class ChartVisibilityButton extends StatelessWidget {
```
[the remark this answers is not in the document]
Comment (kenzieschmoll) [reply packages/devtools_app/lib/src/shared/common_widgets.dart:505]: done

File: packages/devtools_app/lib/src/shared/common_widgets.dart [hunk 2]
```diff
@@ -273,6 +273,47 @@ class ResumeButton extends StatelessWidget {
   }
 }
 
+class PauseResumeButtonGroup extends StatelessWidget {
```
[the remark this answers is not in the document]
Comment (kenzieschmoll) [reply packages/devtools_app/lib/src/shared/common_widgets.dart:276]: done

File: packages/devtools_app/lib/src/shared/common_widgets.dart [hunk 3]
```diff
@@ -2153,6 +2246,9 @@ class _BlinkingIconState extends State<BlinkingIcon> {
   }
 }
 
+// TODO(): investigate if we can modify this widget to be a
```
[the remark this answers is not in the document]
Comment (kenzieschmoll) [reply packages/devtools_app/lib/src/shared/common_widgets.dart:2249]: done

**Pull request merged**